script/review-entrypoint.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/sh
+
 set -e
 yarn install --production=false
 npm run fetch-drupal-cache
 npm run build -- --buildtype localhost --api='${API_URL}' --host='${WEB_HOST}' --port='${WEB_PORT}'
-npm run heroku-serve -- build/localhost -p 3001
+
+if [ -d "../content-build" ]
+then
+  cd ../content-build
+  yarn install --production=false
+  npm run build -- --buildtype localhost --api='${API_URL}' --host='${WEB_HOST}' --port='${WEB_PORT}'
+  npm run heroku-serve -- build/localhost -p 3002
```

Comment:
Any reason why this can't be 3001? It should be the only HTTP server running, so I wouldn't expect any collision in port numbers. I think tests may be expecting the site to be served locally on 3001 as well.

Reply:
The `content-build` repo is expecting to be served on 3002 by default. Serving it on 3001 affects the URL mappings. I tested `content-build` on 3001 and some pages didn't load properly. There are some [Cypress fixtures](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/layouts/tests/vamc/fixtures/health_care_local_facility_no_phone_mental.json#L140) that hard-code `localhost:3001` that will need to be updated.

@timwright12 This also means we'll probably need to update the ports in [docker-compose.review.yml](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docker-compose.review.yml) once we make the switch, since we'd be serving on 3002.